frontend/src/components/maps/leaflet/InfoSlideOut/HeaderActions.tsx

```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components';
+import * as React from 'react';
+import { Row } from 'react-bootstrap';
+import { IBuilding, IParcel, PropertyTypes } from 'actions/parcelsActions';
+import { useLeaflet } from 'react-leaflet';
+import { MAX_ZOOM } from 'constants/strings';
+import { Link, useLocation } from 'react-router-dom';
+import queryString from 'query-string';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Claims from 'constants/claims';
+
+const LinkMenu = styled(Row)`
+  background-color: #f2f2f2;
+  height: 35px;
+  width: 341px;
+  margin-left: -10px;
+  font-size: 14px;
+  font-family: ProximaNova, sans-serif;
+  padding: 10px;
+  a {
+    padding: 0 10px;
+    color: #1a5a96;
+  }
+`;
+
+const VerticalBar = styled.div`
+  border-left: 6px solid rgba(96, 96, 96, 0.2);
+  height: 18px;
+  margin-left: 1%;
```

Comment:
a little surprised to see % margins, any particular reason for this? generally percentages are used when the size relative to the parent is important.

Reply:
I just copied this code from elsewhere, it doesn't really affect it much so I'll pull it out